guides/v2.2/install-gde/prereq/security.md

```diff
@@ -1 +1,66 @@
-../../../v2.1/install-gde/prereq/security.md
\ No newline at end of file
+---
+group: installation-guide
+subgroup: Prerequisites
```

Comment:
The following frontmatter entries are obsolete and should be deleted:

- `subgroup`
- `menu_title`
- `menu_order`